launch/doc/source/conf.py

```diff
@@ -55,6 +55,7 @@
 # ones.
 extensions = [
     'sphinx.ext.autodoc',
+    'sphinx.ext.napoleon',
```

Comment:
Why do we need to add this extension?

Reply:
Ah nope forgot to remove it after experimenting.
[8273280](https://github.com/ros2/launch/pull/697/commits/8273280a0080c624fd53e74ec4838eb70e254825)